examples/kv-cmf/kv_replica_msgs.cmf

```diff
@@ -0,0 +1,62 @@
+# Byzantine fault-tolerant K/V store messages/commands for the open source state machine example.
+#
+# Note KV effectively implements a versioned key-value store supporting
+# only keys and values that are byte strings.
+
+Msg KVReadRequest 3 {
+
+    # Note an KVReadRequest with read version 0 indicates that the latest
+    # version should be read.
+    uint64 read_version
+
+    list bytes keys
+}
+
+Msg KVWriteRequest 4 {
```

Comment:
Why are we not using cmf from existing client?

Reply:
We do not want to make this example/demo complex for the user to understand that why we have used the existing cmf key-value message. It will be updated according to requirements.